cachematrix.R

```diff
@@ -1,15 +1,62 @@
-## Put comments here that give an overall description of what your
-## functions do
-
-## Write a short comment describing this function
-
```

Comment:
Hello, your assignment looks fine, there is no mistake in any of the codes.
Thank you
All the best